### PR TITLE
remove default flow base url

### PR DESCRIPTION
**Pull request by ajanczak**

### CODE REVIEW

File: postman-generator/src/test/scala/generator/PostmanCollectionGeneratorSpec.scala [hunk 1]
```diff
@@ -77,7 +77,7 @@ class PostmanCollectionGeneratorSpec extends WordSpec with Matchers {
           |  },
           |  "variable" : [ {
           |    "type" : "string",
-          |    "value" : "https://api.flow.io",
+          |    "value" : "",
```
Comment (anadoba) [inline postman-generator/src/test/scala/generator/PostmanCollectionGeneratorSpec.scala]: it would be good to add a `baseUrl` to our `trivialService` that is used in this test case.
So the value is not empty.

You can also add a short test case for the empty case if you think it'd be beneficial.
test on a parsed model, not the string assertions
(e.g. `collection.variable shouldEqual Seq(Variable(_, "")))`

**Comment by anadoba** [comment]
there is a TODO comment left by me in `generator/PostmanItemBuilderSpec.scala:206`
Initially, I thought that we will remove this variable completely.
However, your PR preserves this variable and just sets it with `service.baseUrl` thing.
I think your solution is even better :)
So you can just delete my TODO mentioned above in this PR

**Review by anadoba — APPROVED** [review]
